libwacom/libwacom.h

```diff
@@ -309,6 +309,10 @@ typedef enum {
 	WACOM_STATUS_LED_DIAL2		= 5,
 } WacomStatusLEDs;
 
+typedef enum {
+	IGNORE_ALIASES	= 0,
+	ONLY_ALIASES	= 1,
+} AliasStatus;
```

Comment:
This one must not be in `libwacom.h`, otherwise it becomes public API. Just putting it into `libwacom-database.c` is enough, it's not used anywhere else.